docs/reference/query-dsl/exists-query.asciidoc

```diff
@@ -6,11 +6,7 @@ Returns documents that have at least one non-`null` value in the original field:
 [source,js]
 --------------------------------------------------
 {
-    "constant_score" : {
-        "filter" : {
-            "exists" : { "field" : "user" }
-        }
-    }
+        "exists" : { "field" : "user" }
```

Comment:
will do. I just saw I opened the PR agains 2.0 from the online docs page, should this still go on 2.0 since it is already out? Otherwise I'd only do this for 2.1 and upwards.